### DISCUSSION TITLE
Automatically block Xunlei, QQ, Baidu, Xfplay, DLBT and offline downloaders. Can it be turned off?

### DISCUSSION THREAD

**Discussion by fgr178707** [c0re100/qBittorrent-Enhanced-Edition#490] [Q&A]
Automatically block Xunlei, QQ, Baidu, Xfplay, DLBT and offline downloaders. Can it be turned off?
自动封禁迅雷、QQ、百度、Xfplay、DLBT和离线下载器 能关闭吗

**Comment by largithub123** [comment]
Give a switch, it is off by default, you can choose to open it, so that other clients can download it, it will be used when making seeds

**Reply by fgr178707** [reply]
I'm asking which one I should turn on if I disable it.
Auto Ban Bitto rrent Media Player Peer?
Auto Ban Unknown Peer from China?
